runtime/src/runtime/mod.rs

```diff
@@ -142,7 +144,63 @@ pub trait Runtime: Primitives + Verifier + RuntimePolicy {
         method: MethodNum,
         params: Option<IpldBlock>,
         value: TokenAmount,
-    ) -> Result<Option<IpldBlock>, ActorError>;
+    ) -> Result<Option<IpldBlock>, ActorError> {
+        match self.send_generalized(to, method, params, value, None, SendFlags::empty()) {
+            Ok(ret) => {
+                if ret.exit_code.is_success() {
+                    Ok(ret.return_data)
+                } else {
+                    Err(ActorError::checked(
+                        ret.exit_code,
+                        format!(
+                            "send to {} method {} aborted with code {}",
+                            to, method, ret.exit_code
+                        ),
+                        ret.return_data,
+                    ))
+                }
+            }
+            Err(err) => Err(match err {
+                // Some of these errors are from operations in the Runtime or SDK layer
+                // before or after the underlying VM send syscall.
+                ErrorNumber::NotFound => {
+                    // This means that the receiving actor doesn't exist.
+                    // TODO: we can't reasonably determine the correct "exit code" here.
```

Comment:
These TODOs are lifted from the previous implementation in `fvm.rs`. I'm not sure whether they're worth keeping around.

Reply:
I agree with dropping the TODOs, they're not useful.